scripts/verify-replace.sh

```diff
@@ -16,26 +20,57 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
+    STDOUTFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-output.XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="$(basename ${repo}) "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies 2> ${ERRFILE} 1> ${STDOUTFILE}
```

Comment:
we need to print the copy of the output also to the terminal. There are two reasons:
1. important for local execution to follow on the progress of the execution. Otherwise, it just stays only with `Initiating 'go mod replace' of current toolchain common version in dependent repos` and stays silent
2. To see the whole context of the potential erros. If the regex selection of the errors doesn't include everything, then it's good to scroll up and see the whole output.